### DISCUSSION TITLE
Starting Teleport fails - Unable to verify HTTPS certificate chain

### DISCUSSION THREAD

**Discussion by pluet24** [gravitational/teleport#11459] [Q&A] [answered]
Currently I am trying to install Teleport locally on my network. For this purpose, I have issued a certificate chain and a private key to my internal CA. Now I get the following message when executing the following command.

![grafik](https://user-images.githubusercontent.com/100721414/160139151-7f471337-451b-4d04-801a-8b036ac5c133.png)

I used the original Teleport instructions for creating the teleport.yaml.

![grafik](https://user-images.githubusercontent.com/100721414/160139594-aa1caee7-a5d5-4a09-b6f8-8af24cd70658.png)

Do I have to store a root CA for the Teleport server and if so, where is this storage location defined?

Thanks in advance!

**Comment by webvictim** [comment, marked as answer]
You can specify the path to your root CA with the `SSL_CERT_FILE` environment variable.

Something like `SSL_CERT_FILE=/path/to/root-ca.pem sudo teleport start` should work.

**Reply by webvictim** [reply]
You can also install the root CA to the system trust store (which is used by Go and thus read by Teleport):

```
cp /path/to/root-ca.pem /usr/local/share/ca-certificates
sudo update-ca-certificates
```

This should negate the need to use the `SSL_CERT_FILE` variable.

**Reply by pluet24** [reply]
Thanks for your reply. That worked for me!

**Reply by webvictim** [reply]
No problem!
